docs/integration_resources/resources/usd.md

```diff
@@ -0,0 +1,29 @@
+# USD
+
+Shift has developed a specialized catalog and utility plugins for using the USD API in Shift. 
+
+* [Shift_USD](https://github.com/Inbibo/Shift_USD)
+
+## Installation
+
+To start working with the USD API in Shift, please follow the following steps:
+
+* Download the necessary dependencies (Python & USD).
+* Clone the [Shift_USD](https://github.com/Inbibo/Shift_USD) repository into the preferred folder.
+* Add the path to the Shift_USD directory to the `PYTHONPATH` environment variable.
+* Add the path to the Shift_USD directory to the `SHIFT_CATALOG_PATH` environment variable.
+* Add the path of the *plugin.json* file found in the Shift_USD directory to the `SHIFT_PLUGIN_PATH` environment variable.
+
+## Dependencies
+| **Dependency**                                           | **Version** |
+| -------------------------------------------------------- | ----------- |
+| [Python](https://www.python.org/download/releases/3.0/)  | > 3.7       |
+| [USD](https://graphics.pixar.com/usd/release/index.html) | > 21.8      |
+
+<!-- ##Plugins
+
+This section is reserved to USD Plugins information 
+
+### Examples
+This section is reserved to an example video of how to use the Python Script node.
+ -->

```

Comment:
Do you want to add a video explaining how the USD Outliner plugin works or just a gif and some text?

Reply:
Both. Ideally in the future most sections in the documentation will have a video too apart from the written content

Reply:
can we have a TODO in here to recall adding that?

Reply:
Done
https://github.com/Inbibo/shift_docs/issues/43